<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompilerSelectingDialog</class>
 <widget class="QDialog" name="CompilerSelectingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Compiler's Location</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="gccLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 10pt;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string notr="true">gcc</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="gccPath"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="gccSelectButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="gppPath"/>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="gppSelectButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fpcLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 10pt;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string notr="true">fpc</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="fpcPath"/>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="fpcSelectButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="gppLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 10pt;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string notr="true">g++</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CompilerSelectingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CompilerSelectingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
